Ranking Criteria updates (#2453)
ppy/osu-wiki: wiki/Ranking_Criteria/en.md

All forms of Artist/Title standardisation apply to both the `Romanised` and `Unicode` fields, excluding standardised whitespaces for full-width characters.

- **Logos should not be used as references for capitalisation of titles or artists.** Because logos are often stylised, apply standard capitalisation unless other textual metadata supports it.
- **Tracks created by artists belonging to doujin circles should list the circle's name as the main artist.** The exception to this is when the artist(s) of a given track is well-known enough by their own name. In this case, the specific artist name(s) may be used instead.
- **If the same song exists in the Ranked section already, the metadata should be followed unless it breaks other rules in the Ranking Criteria or the Official Sources state something completely different.**
- **Single symbols should be romanised so that they have leading and trailing whitespaces, unless the symbol itself is not commonly requiring whitespaces in English.** This may be ignored if the artist purposefully uses special characters that ignore their common usages.

> - **If the song is shortened for reasons other than a remix, a cover, or an official cut, it should include `(Cut Ver.)` at the end of its title.** This is to clearly signify shortened songs from their full versions. Songs that are insignificantly shortened, songs that are shortened in ways that nearly match their original versions, and songs that are a full loop of a looping track are exempt. A song that has been extended either through looping or slowing down segment(s) should add `(Extended Ver.)` at the end of its title.

Reviewer: ```suggestion
- **If the song is shortened for reasons other than a remix, a cover, or an official cut, it should include `(Cut Ver.)` at the end of its title.** This is to clearly signify shortened songs from their full versions. Songs that are insignificantly shortened, shortened in ways that nearly match their original versions, or cut to a full loop of a looping track are exempt. A song that is extended either through looping or slowing down segments should include `(Extended Ver.)` at the end of its title.
```

- **Active hit sounds:** Hit sounds that reach their peak impact exactly when they are clicked.
- **Passive hit sounds:** Hit sounds that are not designed to match a player's exact clicking, such as ambient sounds.
- **Storyboarded hit sounds:** Hit sounds played through storyboard coding.
- **Cut:** A song file that has had a portion of audio removed to shorten the beatmapset's play time.

### Rules

> - **A beatmapset's audio file must use the `.mp3` file format and have a constant bit rate no greater than 192kbps.**

Reviewer: the wording here accidentally(?) disallows VBR audio entirely
```suggestion
- **A beatmapset's audio file must use the `.mp3` file format and have a constant or average bit rate no greater than 192kbps.**
```

- **A song's audio file must be of reasonable quality.** Try to find the highest quality source file available rather than ripping a file from a streaming video website. Songs should be normalized to their original release volumes and should not be encoded to a bit rate higher than their original files.
- **If you do not beatmap the last 20% of your beatmapset's audio file, it must be cut.** The intro time is not included. This does not apply if more than 20% of the outro is occupied by a storyboard/video.
- **Beatmaps must be hit sounded.** While every game mode has its own techniques, effective hit sounds in all modes accent the most important parts of the music.
- **All clicked objects must have audible active hit sounds or active hitnormal samples.** osu!mania beatmaps are exempt from this because of the mode's rhythm construction.
- **Active hit sounds must use the `.wav` or `.ogg` file formats.** `.ogg` files must use a bit rate of 128kbps or higher. `.mp3` files have slight delays, and therefore are reserved only for longer passive hit sounds, such as ambient noises. 

> - **Applause files and other passive hit sounds must use the `.mp3` or `.ogg` file formats.** These files usually have long durations and `.wav` files are unnecessarily large in comparison. 

Reviewer: ```suggestion
- **Applause files and other passive hit sounds must use the `.mp3` or `.ogg` file formats.** These files usually have long durations and `.wav` files are unnecessarily large in comparison.
```
